Kiosk/Views/InkRecognizerExplorer/DualCanvas.xaml

```diff
@@ -0,0 +1,177 @@
+                                <InkToolbarPencilButton x:Name="pencil"/>
+                                <InkToolbarEraserButton x:Name="eraser" IsClearAllVisible="False"/>
+                                <InkToolbarCustomToggleButton x:Name="touchButton" Click="TouchButton_Click" ToolTipService.ToolTip="Touch Writing">
+                                    <SymbolIcon Symbol="{x:Bind Path=TouchWriting}"/>
+                                </InkToolbarCustomToggleButton>
+                            </InkToolbar>
+
+                            <Rectangle Height="24" Width="1" Fill="Gray" />
+                            
+                            <StackPanel Orientation="Horizontal" x:Name="customToolbar" RequestedTheme="Light" >
+                                <InkToolbarCustomToolButton x:Name="undoButton" Click="UndoButton_Click" ToolTipService.ToolTip="Undo">
+                                    <SymbolIcon Symbol="{x:Bind Path=Undo}"/>
+                                </InkToolbarCustomToolButton>
+                                <InkToolbarCustomToolButton x:Name="redoButton" Click="RedoButton_Click" ToolTipService.ToolTip="Redo">
+                                    <SymbolIcon Symbol="{x:Bind Path=Redo}"/>
+                                </InkToolbarCustomToolButton>
+                                <InkToolbarCustomToolButton x:Name="clearButton" Click="ClearButton_Click" ToolTipService.ToolTip="Clear All">
+                                    <SymbolIcon Symbol="{x:Bind Path=ClearAll}"/>
+                                </InkToolbarCustomToolButton>
+                            </StackPanel>
+                        </StackPanel>
+                    </Viewbox>
+
+                    <Viewbox Grid.Column="1" VerticalAlignment="Stretch" Margin="0,5,10,0">
+                        <StackPanel Orientation="Horizontal" >
+                            <ComboBox x:Name="languageDropdown" SelectionChanged="LanguageDropdown_SelectionChanged" SelectedIndex="3" Width="175" RequestedTheme="Light">
+                                <ComboBoxItem Name="zhCN" Content="Chinese (Simplified)"/>
+                                <ComboBoxItem Name="zhTW" Content="Chinese (Traditional)"/>
+                                <ComboBoxItem Name="enGB" Content="English (UK)"/>
+                                <ComboBoxItem Name="enUS" Content="English (US)"/>
+                                <ComboBoxItem Name="frFR" Content="French"/>
+                                <ComboBoxItem Name="deDE" Content="German"/>
+                                <ComboBoxItem Name="itIT" Content="Italian"/>
+                                <ComboBoxItem Name="jaJP" Content="Japanese"/>
+                                <ComboBoxItem Name="koKR" Content="Korean"/>
+                                <ComboBoxItem Name="ptPT" Content="Portuguese"/>
+                                <ComboBoxItem Name="esES" Content="Spanish"/>
```

Comment:
this is a little strange.  You should use something like SelectedValuePath and specify the actual value rather than create a name that you have to modify in the code behind.  Check out [https://asp-net-example.blogspot.com/2017/01/uwp-combobox-displaymemberpath-and.html](https://asp-net-example.blogspot.com/2017/01/uwp-combobox-displaymemberpath-and.html) for instance...